apps/wallet-dashboard/components/dialogs/staking/views/DetailsView.tsx

```diff
@@ -169,6 +191,7 @@ export function DetailsView({
                         text="Stake"
                         onClick={handleStake}
                         fullWidth
+                        disabled={isAtRisk}
```

Comment:
there is a mismatch between enabling and disabling stake button in wallet vs dashboard. The user should be able to stake/delegate to any active validator